src/main/target/CRAZYBEEF3FR/target.h

```diff
@@ -133,12 +133,13 @@
 #define USE_RX_FRSKY_SPI_D
 #define USE_RX_FRSKY_SPI_X
 #define USE_RX_FRSKY_SPI_TELEMETRY
+#define USE_RX_SFHSS_SPI
```

Comment:
If this target is overflowing try uncommenting the `#undef USE_ESC_SENSOR_INFO` above.

Reply:
uncomment
#undef USE_PINIO
#undef USE_PINIOBOX
#undef USE_ESC_SENSOR_INFO